Makefile

```diff
@@ -291,13 +305,25 @@ gardener-down: $(SKAFFOLD) $(HELM)
 	@# cleanup namespaces that don't get deleted automatically
 	kubectl delete ns gardener-system-seed-lease istio-ingress istio-system --ignore-not-found
 
+gardenlet-kind2-up: $(SKAFFOLD) $(HELM)
+	$(SKAFFOLD) run -m gardenlet,provider-local -p kind2
+
+gardenlet-kind2-down: $(SKAFFOLD) $(HELM)
+	$(SKAFFOLD) delete -m gardenlet -p kind2
```

Comment:
Why do we not delete `provider-local` here?

Reply:
`provider-local` isn't actually deployed for the `kind2` profile as it is managed by the controller registration in the garden cluster. So there is no reason to delete it here.

Reply:
Makes sense, thank you!